### PR TITLE
Fixes Title bar visibility issue

### LINKED ISSUE #890: Title bar items not visible
**Describe the bug**

When the app is in the fragment screens, the title bar items are not visible. Since the letters are in white and the title bar is also set to white, so is the problem there.As could be seen clearly only those items which are not white in color are visible.

**Expected behavior**

It should be transparent or some other color so that items and text should be visible.

**Screenshots**
kiwix white title bar 1

kiwix white title bar2
kiwix white title bar3

**Environment**

- Device: Redmi 4
- OS version : Android 7.1.2

**Request**
Please assign me for this issue.

### PR DESCRIPTION
**Pull request by sonusourav**
Fixes #890 

Changes: 

- Overrode the setTheme in Base Activity.
- Defined new style StatusBar style for Base activity.

Screenshots/GIF for the change: 
**Before**
before

**After in day mode**
after

**After in night mode**
after_night

### CODE REVIEW

**Comment by abdulwd** [comment]
Does it work for API < 21?

**Comment by sonusourav** [comment]
No, it does not. So that's why the SDK check is there.

**Comment by abdulwd** [comment]
You should fix this for all APIs.

**Comment by sonusourav** [comment]
@abdulwd Now it works for all devices. Please review it once.

File: app/src/main/java/org/kiwix/kiwixmobile/bookmark/BookmarksActivity.java [hunk 1]
```diff
@@ -101,6 +101,7 @@ protected void onCreate(Bundle savedInstanceState) {
       actionBar.setDisplayHomeAsUpEnabled(true);
       actionBar.setTitle(R.string.menu_bookmarks);
     }
+
```
Comment (mhutti1) [inline app/src/main/java/org/kiwix/kiwixmobile/bookmark/BookmarksActivity.java]: Please don't commit files you don't touch.

**Pull request merged**